.prettierrc

```diff
@@ -1,4 +1,6 @@
 {
   "semi": false,
-  "singleQuote": true
-}
\ No newline at end of file
+  "singleQuote": true,
+  "tabWidth": 2,
+  "printWidth": 120
```

Comment:
The default is `80` and I feel like this is too little. Let me know if you disagree and I will switch it back.